### ISSUE TITLE
Add ENV VAR to expose --safe mode

### ISSUE THREAD

**Issue by knolleary** [node-red/node-red-docker#102]
There is no easy way to run Node-RED in our docker image with safe mode enabled - because that can only be done via command-line arg.

We should provide a way to enable it via env var or other mechanism.

**Comment by Maddin-619** [comment]
There's a way even if it's not very beautiful:
`docker run -it -p 1880:1880 --entrypoint "/bin/bash" nodered/node-red-docker -c 'npm start -- --userDir /data --safe'`
Or in docker-compose file:
```
node-red:
  image: nodered/node-red-docker:latest
  ports:
    - "1880:1880"
  entrypoint: "npm start -- --userDir /data --safe"
```

**Comment by dceejay** [comment]
There are different ways to solve this.

 - [ ] 1) as you suggest with a specific environment variable  - though it should be SAFE_MODE (not SAVE_MODE)
 - [ ] 2) add a general NODE_RED_ARGS environment variable and pass these through to the Node-RED runtime.

The advantage of 2 is that it allows us to pass "any" parameters through including --safe.  
the disadvantage is that users could put the wrong or bad things in and break their runtime (potentially). Are the other command line parameters things that are useful to pass in ?

Discuss....

**Comment by dceejay** [comment]
Fixed by c7bbe2f1fea014fa1559dc8c56c45f23b2abe41c in core of Node-RED
Will be exposed in 1.0.1
